opm/input/eclipse/Parser/ErrorGuard.cpp

```diff
@@ -58,22 +50,40 @@ namespace Opm {
         }
 
         if (!this->error_list.empty()) {
-            const std::regex file_regex("\n\\w*In file(.*)line (.*)");
```

Comment:
Since we remove all uses of `regex` we can also remove the [`<regex>`](https://github.com/OPM/opm-common/blob/7098e3ad9a84882cbbb27c734642e62e131a4cdc/opm/input/eclipse/Parser/ErrorGuard.cpp#L25) header from this translation unit.